frontend/src/components/SeedlotRegistrationProgress/InterimStorage/index.tsx

```diff
@@ -9,28 +10,31 @@ import {
   ComboBox,
   DatePicker,
   DatePickerInput,
-  NumberInput,
   RadioButton,
   RadioButtonGroup,
   Row,
-  TextInput
+  TextInput,
+  FlexGrid
 } from '@carbon/react';
 
-import { useParams } from 'react-router-dom';
 import { ArrowRight } from '@carbon/icons-react';
-import { FilterObj, filterInput } from '../../../utils/filterUtils';
 import Subtitle from '../../Subtitle';
 import InterimStorageRegistration from '../../../types/InterimStorageRegistration';
-import getUrl from '../../../utils/ApiUtils';
+import { FilterObj, filterInput } from '../../../utils/filterUtils';
 import ApiAddresses from '../../../utils/ApiAddresses';
+import getUrl from '../../../utils/ApiUtils';
 import { useAuth } from '../../../contexts/AuthContext';
-import ApplicantInfo from '../../../types/ApplicantInfo';
 import './styles.scss';
 
+const DATE_FORMAT = 'Y-m-d';
```

Comment:
The date format should be using '/' to separate year, month and day. If that doesn't work properly for some reason, maybe try to check with the UX team if it is ok to use the '-'.

Reply:
@Wennio-Oliveira The `DATE_FORMAT` is only used by the date picker component not by moment, I have updated it, check it out

Reply:
Fixed!